src/aou-common/load-envs/load-env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# load-env.sh Loads environment variables for the attached AoU data collection
+# version.
+#
+# Usage: source load-env.sh
+
+# Get the environment variables inside a function so that the set -o options
+# don't impact the user's environment, since this script needs to be sourced to
+# apply the environment variables.
+function get_env_vars() {
+    set -o errexit
```

Comment:
set these outside the scope of this function?

Reply:
I would prefer not to since this file is sourced by bashrc. Moving it out of the function will set those for the user's shell session